src/uoapi/course/prereq.py

```diff
@@ -119,6 +115,27 @@ def __init__(self, prereq_str):
                 if match is not None:
                     if key == "prerequisites":
                         self.prereqs = self.parse_codes(match)
+                        self.parse_status = True
                     break
 
 
+def recursive_process(node, lookup):
```

Comment:
Have we checked what kind of coverage this gives us in terms of how many courses' prerequisites we successfully parse? (Again, just curious, not necessarily requesting a change here.)